lib/routes/tarballs.js

```diff
@@ -10,9 +10,9 @@ r.get('/:scope?/:name/-/:scope2?/:filename/:sha', middleware.auth.read, aw(async
   if (sha.startsWith(filename)) {
     // I have no idea why the client requests this
     // just redirect to the right URL
-    let url = `/${name}/-/${scope2}/${sha}`
-    // TODO: fix scoped packages
-    // if (scope) url = `/${scope}${url}`
+    let _scope = scope ? `/${scope}` : ''
```

Comment:
Great fix, thanks Ian! Can we convert the `let` to a `const` as the value isn't reassigned. Also curious on the underscore naming convention. Are you denoting it's private?